### DISCUSSION TITLE
Don't know if it's a bug, but seems important

### DISCUSSION THREAD

**Discussion by Valporaena** [ArchiveBox/ArchiveBox#839] [General]
So, I was just working with Archivebox web interface - adding/deleting stuff - and after trying to delete a couple of items on the list my whole archive got completely obliterated, showing 0 snapshots as a result. All of the ~10000 snapshots are gone, nothing in the /archivebox/archive directory either. I'm pretty sure I didn't check "check all" box, was just adding/deleting a couple of things. Thank god Archivebox saves the sources list, or I would've been completely screwed! Still really sucks for me and maybe worth looking into for the devs imo.

**Comment by pirate** [comment]
Sorry this happened! I always recommend taking automatic periodic backups of archivebox using ZFS snapshots.

I haven't seen anything similar in the last couple years, so I'm going to close this as stale for now. If this ever happens again let me know. We do have some bugs around Safari's handling of the checked rows, but I've never seen it select all the rows by accident before or since this report.
